api/spot.yaml

```diff
@@ -552,8 +552,12 @@ paths:
     $ref: services/managedInstance/aws/paths/managedIDDeleteVolume.yaml
   /aws/ec2/managedInstance/migration:
     $ref: services/managedInstance/aws/paths/migration-mi.yaml
+  /aws/ec2/managedInstance/migration/bulk:
+    $ref: services/managedInstance/aws/paths/bulk-migration-mi.yaml
```

Comment:
Please use camelcase format for file names (across this PR).
Refer to the following guidelines docs:
https://docs.google.com/document/d/1mMKAsasXl6oSHOfr0mETWjefBPCRPqAKj7ut2Atju_Y/edit
https://docs.google.com/document/d/1TGRKymd3h-fmn9Dy8-V4tGOatnK0iSlkREt59QmOQMU/edit